pairtaghighlighter/src/pair_tag_highlighter.c

```diff
@@ -319,6 +321,9 @@ static void findMatchingTag(ScintillaObject *sci, gint openingBracket, gint clos
     gboolean isTagOpening = is_tag_opening(sci, openingBracket);
     gchar *tagName = get_tag_name(sci, openingBracket, closingBracket, isTagOpening);
 
+    if (!tagName)
```

Comment:
I'm not familiar with this plugin's code, is it acceptable to silently fail here?